src/auth/services/auth-service.spec.ts

```diff
@@ -24,10 +24,12 @@ describe('auth/', () => {
         authSubject.subscribe(callback);
       });
 
-      addProviders([
-        {provide: FirebaseAuth, useValue: mockFirebaseAuth},
-        AuthService
-      ]);
+      TestBed.configureTestingModule({
```

Comment:
Here's the corresponding [CHANGELOG @rc6](https://github.com/angular/angular/blob/master/CHANGELOG.md#200-rc6-2016-08-31) corresponding to this breaking change